Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanelModifier.swift

```diff
@@ -28,6 +28,9 @@ public extension View {
     ///   - content: A closure that returns the content of the floating panel.
     /// - Returns: A dynamic view with a presentation style similar to that of a sheet in compact
     /// environments and a popover otherwise.
+    @available(iOS 16.0, *)
+    @available(macCatalyst 16.0, *)
+    @available(visionOS, unavailable, message: "Use 'floatingPanel(attributionBarHeight:selectedDetent:horizontalAlignment:isPresented:maxWidth:_:)' instead.")
```

Comment:
Since using colors isn't really encouraged in visionOS and the default color for this modifier is `systemBackground` which doesn't exist in visionOS it doesn't seem to make sense to offer this view modifier with a background color parameter for visionOS. So instead I added a very similar view modifier below that is available on visionOS and it doesn't include the `backgroundColor` parameter.
![Screenshot 2024-09-24 at 11 56 42 AM](https://github.com/user-attachments/assets/6a6b8ad9-896d-4a27-8aa9-59276b00e1f8)